src/styles/card.scss

```
@@ -0,0 +1,83 @@
+.card {
+  position: relative;
+  height: 403px;
+  width: 200px;
+  margin: 0 48px 50px 0;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  &__last {
+    margin-right: 30px;
+  }
+  &__img {
+    height: 134px;
+    width: 160px;
+    margin: 30px 19px 41px;
+  }
```

Розділяй селекторами відступами, тоді їх буде простіше читати ?
```suggestion
  &__last {
    margin-right: 30px;
  }
  
  &__img {
    height: 134px;
    width: 160px;
    margin: 30px 19px 41px;
  }
```